scheduler/hooks/retrieve_after_store.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Usage: bash retrieve_after_store.sh <UUID>
+# expected installed env binaries: mktemp, curl, jq
+# env vars used:
+# DEALBOT_CONTROLLER_ENDPOINT
+
+curl -o /tmp/dealbot.$1 $DEALBOT_CONTROLLER_ENDPOINT/tasks/$1\?parsed
+
+if [ "$(cat /tmp/dealbot.$1 | jq '.StorageTask == null')" = "true" ]; then
```

Comment:
you can avoid the cat, like:

    if [ "$(jq '.StorageTask == null' /tmp/dealbot.$1)" = "true" ]; then

this is bash, so you could also use the safer and more intuitive `[[` builtin, which e.g. saves you from needing quotes for safety against word splitting:

    if [[ $(jq '.StorageTask == null' /tmp/dealbot.$1) == "true" ]]; then